README.md

```diff
@@ -48,22 +52,31 @@ Tools can be acquired via instructions in [third_party](/third_party/README.md).
 Project is defined using [CMake](https://cmake.org/).
 
 **Windows**:
+
 Generating Visual Studio Project.  EXE and binaries will be in **build/bin/{build_config}**
 
 ~~~~
-mkdir build
-cd build
-cmake {path_to_source_dir} -G "Visual Studio 15 2017 Win64"
+$ mkdir build
+$ cd build
+$ cmake {path_to_source_dir} -G "Visual Studio 15 2017 Win64"
 ~~~~
 
 **Linux**:
 
 Executable and binaries will be in **build/bin**
 
 ~~~~
-mkdir build
-cd build
-cmake {path_to_source_dir}
-make
+$ mkdir build
+$ cd build
+$ cmake {path_to_source_dir}
+$ make
+~~~~
+
+**General**:
+
+If you've made modifications to the specification, you can also run a custom `generate` target prior to building.
```

Comment:
I think it's better to move this section to be one of the first ones, so this will be something that the user will try using first instead of doing this the hard way (by figuring out which script should be run and with which params, see Source Code Generation section and the linked README).